docker-compose.yml

```
@@ -59,9 +59,18 @@ services:
   memcached:
     <<: *restart_policy
     image: "memcached:1.5-alpine"
+    healthcheck:
+      test: echo stats | nc 127.0.0.1 11211
+      interval: 10s
+      retries: 60
   redis:
     <<: *restart_policy
     image: "redis:5.0-alpine"
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
```

Any reason why we cannot do 
```suggestion
      test: redis-cli ping
```

like above?